addons/mail/static/src/core/common/message.js

```diff
@@ -428,16 +428,6 @@ export class Message extends Component {
         this.popover.open(ev.target, { message });
     }
 
-    onClickFailure(ev) {
```

Comment:
`onClickFailure` appears to be patched in `snailmail` too with a remaining call to super (this function).